app/Http/Controllers/EventController.php

```diff
@@ -560,7 +560,10 @@ public function icalCalendar($personal_key = null)
                     if ($event->activity->isHelping($user)) {
                         $status = 'Helping';
                         $info_text .= ' You are helping with this activity.';
-                    } elseif ($event->activity->isParticipating($user) || $event->hasBoughtTickets($user)) {
+                    } elseif ($event->activity->isOnBackupList()){
```

Comment:
The isOnBackupList function takes a user as an argument, so this will not work

Reply:
PHPStorm did a little trolling :rage::rage::rage: 

Commit 366adf3 should fix this.